.htmltest.yml

```diff
@@ -0,0 +1,30 @@
+IgnoreDirectoryMissingTrailingSlash: true
+CheckDoctype: false
+#CheckExternal: false
+IgnoreDirs:
+- "docs/0.2."
+- "docs/0.3."
+- "docs/0.4."
+- "docs/0.5."
+- "docs/0.6."
+- "docs/0.7."
+- "docs/0.8."
+- "docs/0.9."
+- "docs/0.10."
+- "docs/0.11."
+- "docs/0.12."
+- "docs/0.13."
+IgnoreURLs:
+- "github.com/keptn/keptn/issues"
+- "github.com/keptn/keptn/commit"
```

Comment:
Why are we ignoring those URLs? I think they should be checked! It's easy to link a wrong issue / commit

Reply:
there is no nice way around handling the request limit of GitHub so overtime those will return a 429 - as there are simply too many. my simple and naive approach is to just ignore them for now. as they are not as crucial as internal links within the dokumentation

Reply:
I see :D makes sense